operator_ui/src/Private.js

```diff
@@ -109,7 +109,7 @@ class Private extends React.Component {
                 />
                 <PrivateRoute
                   exact
-                  path="/jobs/:jobSpecId/definition"
+                  path="/jobs/:jobSpecId/json"
```

Comment:
This also needs to `mv operator_ui/src/pages/jobs/_jobSpecId_/definition.js` -> `operator_ui/src/pages/jobs/_jobSpecId_/json.js`